### PR TITLE
Cleanup of css and fonts, fix app.json

### PR DESCRIPTION
**Pull request by bjedrzejewski**

### CODE REVIEW

**Comment by bjedrzejewski** [comment]
All the stuff here gets built anyway including app.json

File: gruntfile.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* jshint node: true*/
 'use strict';
 module.exports = function(grunt) {
-    var target = grunt.option('target') || 'http://localhost:5000',
+    var target = grunt.option('target') || 'http://owennw.github.io/OpenFinD3FC',
         port = process.env.PORT || 5000,
```
Comment (owennw) [inline gruntfile.js]: Is this the bit that determines the settings in the app.json?

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,6 @@
 node_modules
-public/assets/css/style.css*
 public/OpenFinD3FC.zip
-.idea
\ No newline at end of file
+.idea
+public/assets/fonts
+public/assets/css
+public/app.json
```
Comment (owennw) [inline .gitignore]: why not just ignore `public/assets/**/*`?

File: gruntfile.js (hunk 1 above)
Comment (bjedrzejewski) [inline gruntfile.js]: Yes, so we can have another issue so that we can build either local or public

**Comment by owennw** [comment]
Given we're now ignoring more of the public/assets folder it would make sense to remove those files too.

File: .gitignore (hunk 2 above)
Comment (bjedrzejewski) [inline .gitignore]: That will ignore all the files. I hope to have a directory structure there once the app gets bigger?

**Comment by bjedrzejewski** [comment]
So you want to move your angular stuff to src?

**Comment by bjedrzejewski** [comment]
And fully build public?

File: .gitignore (hunk 2 above)
Comment (owennw) [inline .gitignore]: It will only ignore the files in the assets folder, and we don't have anything of our own in there at the moment right?

Comment (bjedrzejewski) [inline .gitignore]: My bad, you are right

**Comment by owennw** [comment]
We can do that, but I think that's orthogonal to what's in this PR. If we're adding files to the .gitignore because they get built then there's no point leaving historic versions of them in the repo.

File: .gitignore (hunk 2 above)
Comment (bjedrzejewski) [inline .gitignore]: Can you commit that fix?

Comment (owennw) [inline .gitignore]: Sure.

Comment (bjedrzejewski) [inline .gitignore]: Actually I will do it

**Comment by bjedrzejewski** [comment]
@owennw when it passes you can merge I hope :)

**Comment by owennw** [comment]
The app.json should be deleted, and the d3fc-showcase.html should be deleted and ignored.

In general anything in the .gitignore should not be in the repo: if it is it should be removed.

**Comment by owennw** [comment]
Looks good. A bit frustrating that running the grunt serve with a target changes the app.json file, but we'll just remember to revert it. Can you squash these commits then I'll merge?

**Comment by bjedrzejewski** [comment]
great, will do now

**Comment by bjedrzejewski** [comment]
@owennw after CI, ready to merge

**Pull request merged**